tests/filesTest.js

```diff
@@ -83,9 +84,7 @@ describe('Main: Currently testing files management,', function () {
                     dProp.appendElement('d:resourcetype', '', dResourcetype => {
                       dResourcetype.appendElement('d:collection', '', '')
                     })
-                      .appendElement('d:quota-used-bytes', '', MatchersV3.number(55))
-                      .appendElement('d:quota-available-bytes', '', MatchersV3.number(3))
-                      .appendElement('d:getetag', '', MatchersV3.regex('"[a-z0-9]+"', '"5f8d0ce8c62b5"'))
+                      .appendElement('d:getetag', '', MatchersV3.string('&quot;5f8d0ce8c62b5&quot;'))
```

Comment:
I would suggest to leave regex for etags, to make sure the etags are valid, meaning are quoted `"`